addon/components/polaris-text-field.js

```diff
@@ -422,7 +422,7 @@ export default Component.extend(ContextBoundTasksMixin, ContextBoundEventListene
   }).readOnly(),
 
   addValueListener() {
-    this.addEventListener('keyup', () => {
+    this.addEventListener('input, textarea', 'keyup', () => {
```

Comment:
should we prefer `this.element.querySelector('input')`....not sure if having just `input` will add listener to all inputs on the page or not

Reply:
@vladucu It uses `findElement(this.element, selector)`, so I think it's scoped to `this.element` of the component.

https://github.com/ember-lifeline/ember-lifeline/blob/master/addon/mixins/dom.js#L105

Reply:
sweet, thx for checking